dnf/plugins/distrosync/distrosync.plugin

```diff
@@ -0,0 +1,11 @@
+[Plugin]
+Module = command_distro-sync
+Embedded = dnf_command_distrosync_register_types
+Name = distro-sync
+Description = Upgrade/downgrade packages to match versions in repositories
+Authors = Neal Gompa <ngompa13@gmail.com>
+License = GPL-2.0+
+Copyright = Copyright © 2021 Neal Gompa
+X-Command-Syntax = distro-sync [PACKAGE…]
+X-Alias-Name = distribution-synchronization
```

Comment:
Please replace with `dsync`. Compatibility with DNF.

According to the DNF documentation, the alias `distribution-synchronization` is deprecated.
```
Distro-Sync Command
       Command: distro-sync
       Aliases: dsync
       Deprecated aliases: distrosync, distribution-synchronization
```